pigpio.js

```diff
@@ -292,6 +303,92 @@ class Notifier {
 
 module.exports.Notifier = Notifier;
 
+/* ------------------------------------------------------------------------ */
+/* Waves                                                                    */
+/* ------------------------------------------------------------------------ */
+
+Gpio.prototype.gpioPulse = pigpio.gpioPulse;
+
+Gpio.prototype.waveClear = function () {
```

Comment:
Please don't used the syntax `Gpio.prototype.waveClear = function () {...}`, add methods to class Goio instead.

Reply:
Corrected methods' style declaration in abe1d13